Gemfile.lock

```diff
@@ -83,30 +83,31 @@ GEM
       rake (>= 10.4, < 14.0)
     apipie-rails (0.5.18)
       rails (>= 4.1)
-    arbre (1.2.1)
-      activesupport (>= 3.0.0)
-    ast (2.4.0)
+    arbre (1.3.0)
+      activesupport (>= 3.0.0, < 6.1)
+      ruby2_keywords (>= 0.0.2, < 1.0)
+    ast (2.4.1)
     aws-eventstream (1.1.0)
-    aws-partitions (1.326.0)
-    aws-sdk-core (3.98.0)
+    aws-partitions (1.381.0)
+    aws-sdk-core (3.109.1)
       aws-eventstream (~> 1, >= 1.0.2)
       aws-partitions (~> 1, >= 1.239.0)
       aws-sigv4 (~> 1.1)
       jmespath (~> 1.0)
-    aws-sdk-kms (1.33.0)
-      aws-sdk-core (~> 3, >= 3.71.0)
+    aws-sdk-kms (1.39.0)
```

Comment:
### [aws-sdk-kms](https://github.com/aws/aws-sdk-ruby)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 1.33.0 → 1.39.0

[_[change-log](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms/CHANGELOG.md), [source-code](https://github.com/aws/aws-sdk-ruby/tree/master/gems/aws-sdk-kms)_]





<details>
<summary>Commits</summary>

A change of **94** commits. See the full changes on [the compare page](https://github.com/aws/aws-sdk-ruby/compare/v1.33.0...v1.39.0).

These are the first 10 commits:
- (1850c6f) [fix ec2-client private_ip_addresses parameter](https://github.com/aws/aws-sdk-ruby/commit/1850c6f8af12604367211e127106a785291e035c)
- (039bcb0) [Merge pull request #456 from kachina/fix-ec2-client](https://github.com/aws/aws-sdk-ruby/commit/039bcb009417658ccc5fef3bba5a56db7eb98b76)
- (6402970) [Support uri unsafe characters in proxy_uri](https://github.com/aws/aws-sdk-ruby/commit/640297066fff98d248ba957e85858b921e25f1e1)
- (a1c0197) [Corrected an issue in the EC2 credential provider around exp…](https://github.com/aws/aws-sdk-ruby/commit/a1c0197977560bc72d2eb0a876daa63648b2d985)
- (3648a18) [Resolved an issue with StackResources.](https://github.com/aws/aws-sdk-ruby/commit/3648a181c7161ff0799f5a24560ded5a6aad0e6b)
- (7aafeff) [Updated AWS::CloudFormation::Client API docs.](https://github.com/aws/aws-sdk-ruby/commit/7aafeff6228b48ccf2c43b8764e0183eb7ca6bf7)
- (fa46f5c) [Added support for the Amazon Route53 2013-04-01 API version.](https://github.com/aws/aws-sdk-ruby/commit/fa46f5c97511cbab7bb587c771407b0a93b2c5c2)
- (c199fcd) [Replace UUIDTools with SecureRandom.uuid](https://github.com/aws/aws-sdk-ruby/commit/c199fcde1c22b9416d8748521eac4d7e1bc2df03)
- (47778ab) [Backport SecureRandom.uuid for Ruby 1.8.7](https://github.com/aws/aws-sdk-ruby/commit/47778abf5234255f4eeb73f4c5e070913739fddb)
- (4ec564c) [Added support to S3::Client#copy_object for the Content-Enco…](https://github.com/aws/aws-sdk-ruby/commit/4ec564cdadef292f0a9daee64270a580c9268051)


</details>